### PR TITLE
Improve usage of `IMPORT FROM SELECT`

### PR DESCRIPTION
**Pull request by lance6716**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-import-into.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 - `IMPORT INTO ... FROM SELECT` 不支持使用 `SHOW IMPORT JOB(s)` 和 `CANCEL IMPORT JOB <job-id>` 等任务管理语句。
 - TiDB [临时目录](/tidb-configuration-file.md#temp-dir-从-v630-版本开始引入)需要有足够的空间来存储整个 `SELECT` 语句查询结果（暂不支持设置 `DISK_QUOTA` 选项）。
 - 不支持使用 [`tidb_snapshot`](/read-historical-data.md) 导入历史数据。
+- 由于 `SELECT` 子句语法复杂，`IMPORT INTO` 的 `WITH` 参数可能与其干扰导致解析时报错。建议为复杂 `SELECT` 创建视图后，使用 `IMPORT INTO ... FROM SELECT * FROM view_name` 导入。或者使用括号明确 `SELECT` 子句的范围，例如 `IMPORT INTO ... FROM (SELECT ...) WITH ...`
```
Comment (D3Hunter) [inline sql-statements/sql-statement-import-into.md:47]:
```suggestion
- 由于 `SELECT` 子句语法复杂，`IMPORT INTO` 的 `WITH` 参数可能与其干扰导致解析时报错，比如 `GROUP BY xxx [WITH ROLLUP]` 等。建议为复杂 `SELECT` 创建视图后，使用 `IMPORT INTO ... FROM SELECT * FROM view_name` 导入。或者使用括号明确 `SELECT` 子句的范围，例如 `IMPORT INTO ... FROM (SELECT ...) WITH ...`
```

Comment (D3Hunter) [reply sql-statements/sql-statement-import-into.md:47]: 其实用 CTE 也行

Comment (lance6716) [reply sql-statements/sql-statement-import-into.md:47]: 这太高端了，我不会

Comment (D3Hunter) [reply sql-statements/sql-statement-import-into.md:47]: https://github.com/pingcap/tidb/blob/3adc71cc71bd301fb38bfd510b5e39734df336fc/pkg/parser/ast/dml_test.go#L587-L588

File: sql-statements/sql-statement-import-into.md [hunk 2]
```diff
@@ -44,6 +44,7 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 - `IMPORT INTO ... FROM SELECT` 不支持使用 `SHOW IMPORT JOB(s)` 和 `CANCEL IMPORT JOB <job-id>` 等任务管理语句。
 - TiDB [临时目录](/tidb-configuration-file.md#temp-dir-从-v630-版本开始引入)需要有足够的空间来存储整个 `SELECT` 语句查询结果（暂不支持设置 `DISK_QUOTA` 选项）。
 - 不支持使用 [`tidb_snapshot`](/read-historical-data.md) 导入历史数据。
+- 由于 `SELECT` 子句语法复杂，`IMPORT INTO` 的 `WITH` 参数可能与其干扰导致解析时报错，比如 `GROUP BY xxx [WITH ROLLUP]` 等。建议为复杂 `SELECT` 创建视图后，使用 `IMPORT INTO ... FROM SELECT * FROM view_name` 导入。或者使用括号明确 `SELECT` 子句的范围，例如 `IMPORT INTO ... FROM (SELECT ...) WITH ...`
```
Comment (hfxsd) [inline sql-statements/sql-statement-import-into.md:47]:
```suggestion
- 由于 `SELECT` 子句语法复杂，`IMPORT INTO` 的 `WITH` 参数可能与其干扰导致解析时报错，比如 `GROUP BY ... [WITH ROLLUP]` 等。建议为复杂 `SELECT` 创建视图后，使用 `IMPORT INTO ... FROM SELECT * FROM view_name` 导入。或者使用括号明确 `SELECT` 子句的范围，例如 `IMPORT INTO ... FROM (SELECT ...) WITH ...`
```

**Review by Frank945946 — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

File: sql-statements/sql-statement-import-into.md [hunk 3]
```diff
@@ -44,6 +44,7 @@ summary: TiDB 数据库中 IMPORT INTO 的使用概况。
 - `IMPORT INTO ... FROM SELECT` 不支持使用 `SHOW IMPORT JOB(s)` 和 `CANCEL IMPORT JOB <job-id>` 等任务管理语句。
 - TiDB [临时目录](/tidb-configuration-file.md#temp-dir-从-v630-版本开始引入)需要有足够的空间来存储整个 `SELECT` 语句查询结果（暂不支持设置 `DISK_QUOTA` 选项）。
 - 不支持使用 [`tidb_snapshot`](/read-historical-data.md) 导入历史数据。
+- 由于 `SELECT` 子句的语法较为复杂，`IMPORT INTO` 的 `WITH` 参数可能会与其冲突，导致解析时报错，例如 `GROUP BY ... [WITH ROLLUP]` 等。建议对复杂的 `SELECT` 语句先创建视图，然后使用 `IMPORT INTO ... FROM SELECT * FROM view_name` 进行导入。或者，可以通过括号明确 `SELECT` 子句的范围，例如 `IMPORT INTO ... FROM (SELECT ...) WITH ...`。
```
Comment (lilin90) [inline sql-statements/sql-statement-import-into.md:47]:
```suggestion
- 由于 `SELECT` 子句的语法较为复杂，`IMPORT INTO` 的 `WITH` 参数可能会与其冲突，导致解析时报错，例如 `GROUP BY ... [WITH ROLLUP]`。建议先对复杂的 `SELECT` 语句创建视图，然后使用 `IMPORT INTO ... FROM SELECT * FROM view_name` 进行导入。或者，可以通过括号明确 `SELECT` 子句的范围，例如 `IMPORT INTO ... FROM (SELECT ...) WITH ...`。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**